### PR TITLE
solving issue #3 and adding a check version feature to adonis-cli

### PR DESCRIPTION
**Pull request by allanfreitas**
This PR resume:

**fix:** standard js fix. 104d4d6

**fix:** the issue #3 25d2edd

**feat:** add a check command to adonis-cli node.js and npm required versions for AdonisJs. 954f3e8

**feat:** add the check command on the adonis-cli new command to stop the install if the check command returns false. 	7493ee6

### CODE REVIEW

**Comment by thetutlage** [comment]
The idea of check looks great to me 😄 Will test out the code

**Comment by allanfreitas** [comment]
Nice @thetutlage  :)

This ideia came to me after the friend that show me AdonisJS tried to use the "adonis new myapp",
and received the error on this image:
adonis-error

We take a while to remember that AdonisJS needs Node.JS>=4.0.0 and npm=>3.0.0 to work :)

File: src/Commands/check.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+const requiredNodeVersion = '>=4.0.0'
+const requiredNpmVersion = '>=3.0.0'
+
+module.exports = function (argv) {
+  const nodejsVersion = process.version.replace('v', '')
+  const npmVersion = execSync('npm -v', {input: 'string', 'encoding': 'utf-8'})
+
+  console.log('-----------AdonisJS requirements check--------------')
+  console.log(`${colors.green('AdonisJS needs Node.JS >= 4.0.0 and npm >= 3.0.0\n')}`)
```
Comment (thetutlage) [inline src/Commands/check.js]: Can you just use the values from `requiredNodeVersion` and `requiredNpmVersion`, instead of using hard coded versions?

**Comment by thetutlage** [comment]
Yup, that makes the entire process so simple. I have left a comment and also can u please create the PR for the `develop` branch?

**Comment by allanfreitas** [comment]
Sorry, I've looking now that I've done on the master branch :(

I will fix that right now.

**Comment by thetutlage** [comment]
No issues, the `develop` branch was not the primary branch. I have fixed that, the future PR's will be targeted towards `develop` by default.

**Comment by allanfreitas** [comment]
I've created a new PR with the right branch on both repos. #5

**Pull request closed without merging**